src/Console/Report/FixReport/TextReporter.php

```diff
@@ -48,7 +48,7 @@ public function generate(ReportSummary $reportSummary): string
             $output .= PHP_EOL;
         }
 
-        return $output.$this->getFooter(
+        return ($reportSummary->shouldPrintFileSummary() ? $output : '').$this->getFooter(
```

Comment:
Maybe better to not fill the `$output` variable in case `$reportSummary->shouldPrintFileSummary()` returns `false` instead of discarding it here?